Remotion/Core/Core/Collections/AutoInitDictionary.cs

```diff
@@ -33,24 +33,24 @@ namespace Remotion.Collections
   public class AutoInitDictionary<TKey, TValue> : IDictionary<TKey, TValue>

```

Comment:
Should we allow `TKey?` here? I think `TKey` should be constrained to `notnull`, but I remember that there were quite some discussions about this.

Reply:
notnull